Formula/harfbuzz.rb

```diff
@@ -16,12 +16,12 @@ class Harfbuzz < Formula
   end
 
   depends_on "glib-utils" => :build
+  depends_on "gobject-introspection" => :build
```

Comment:
Linux may be failing due to all Pythons (e.g. 3.10 from `meson`, 3.9 from `gobject-introspection`) being indirectly installed via build dependencies.

I guess any Python works to run build scripts like macOS Python 3.8

If we want to add test, a test dependency on either Python 3.9 or Python 3.10 could be used as `pygobject3` supports both.